File: exercism/go/raindrops/raindrops.go
/*
Package raindrops converts numbers to rainspeak
*/
package raindrops

import (
	"strconv"
)

// Convert returns the rainspeak for a perticular number
func Convert(number int) string {
	result := ""

	if number%3 == 0 {
		result += "Pling"
	}

	if number%5 == 0 {
		result += "Plang"
	}

	if number%7 == 0 {
		result += "Plong"
	}

	if result == "" {
		result = strconv.Itoa(number)
	}

	return result
}

File: exercism/javascript/raindrops/raindrops.js
export function convert(n) {
  const pling = n % 3 === 0 ? "Pling" : "";
  const plang = n % 5 === 0 ? "Plang" : "";
  const plong = n % 7 === 0 ? "Plong" : "";

  return pling + plang + plong || `${n}`;
}
